src/app/app.module.ts

```diff
@@ -4,16 +4,23 @@ import { BrowserModule } from '@angular/platform-browser';
 import { AppComponent } from './app.component';
 import { AppStoreModule } from './store/app-store.module';
 import { HttpClientModule } from '@angular/common/http';
+import { UsersComponent } from './users/users.component';
+import { ChunkPipe } from './pipes/chunk.pipe';
 
 @NgModule({
     declarations: [
         AppComponent,
+        UsersComponent,
+        ChunkPipe,
     ],
     imports: [
         BrowserModule,
         AppStoreModule,
         HttpClientModule
     ],
+    exports: [
+        ChunkPipe
```

Comment:
netřeba exportovat z rootu

Reply:
to jsou rady s webstormu :D taky rikal ze to je i bez toho funguje proc to sviti cervenym :D